css-three-column-layout/styles.css

```diff
@@ -0,0 +1,68 @@
+.container {
+  margin: auto;
+}
+
+.padding30 {
+  padding-top: 30px;
+  padding-left: 90px;
+  padding-right: 90px;
+}
+
+.set-background-blue {
+  background-color: rgb(26 150 218);
+}
+
+#page-header {
+  color: white;
+  font-family: Lato, sans-serif;
+  align-items: center;
+  justify-content: center;
+  margin-bottom: 0;
+  font-size: 22px;
+}
+
+.row {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.column {
+  flex-basis: calc(100%/3);
+}
+
+.feature-section {
+  color: white;
+  display: flex;
+  flex-direction: column;
```

Comment:
Is this necessary?

Reply:
I have a better understanding of flexbox now and see why this implementation was incorrect.